contracts/contracts/vault/README - Yield Limits.md

```diff
@@ -0,0 +1,52 @@
+# Vault Yield Limits
+
+The vault does not necessarily rebase out all the value that it contains to the token on every rebase. Instead, the vault controls the maximum amount it rebases at one time. 
+
+This limiting behavior protects against donation attacks against other protocols. Some lending platforms don't handle sudden value change well. 
+
+ Secondly, it allows us to smoothly distribute yield over time. One example is that if we do a big sale of airdropped reward tokens, it would be much better if that yield was spread out over a longer period than just one block. Because this rate is controlled by the strategist, it can be set to a high rate to only block donation attacks, but not smooth regular yield, to a middle rate to only smooth extraordinary yield over a some blocks, or a low rate to set the protocols APR to end users, and provide a predictable APY.
```

Comment:
should 
`only smooth extraordinary yield over a some blocks `
be 
`only smooth extraordinary yield over a sum blocks `
or 
`only smooth extraordinary yield over some blocks `